app/boards/shields/murphpad/murphpad.conf

```diff
@@ -1,9 +1,15 @@
-# Uncomment to turn on logging, and set ZMK logging to debug output
-# CONFIG_ZMK_USB_LOGGING=y
+# To enable logging, use the snippet per the documentation.
+# Delaying log output may help catch issues close to startup. Uncomment and adjust if necessary (default is 1000ms)
+#CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=3000
```

Comment:
```suggestion
# CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=3000
```
Minor thing while I'm investigating to see what makes sense, the lack of spaces after the # is annoying me.